tpl/strings/init.go

```diff
@@ -112,6 +112,20 @@ func init() {
 			},
 		)
 
+		ns.AddMethodMapping(ctx.TrimLeft,
+			nil,
+			[][2]string{
+				{`{{ "++Batman++" | strings.TrimLeft "+" }}`, `Batman&#43;&#43;`},
```

Comment:
With the HTML entity encoding thing happening, I would also consider picking another example. It may look fine when rendered somewhere, but it is confusing to look at in the code.